### PR TITLE
Platinum after warm up tb

### PR DESCRIPTION
**Pull request by biankawolo**
Hello! Platinum basic characterization up to T2 for qubits 0,2,4. Qubit 1 is in antisweetspot (too muuch crosstalk) and q3 is near the border of possibilities of QM, cannot be put in its sweetspot. 
q0 T1 = 5.45e4 ns T2 = 1.24e4 ns,
q2 T1 = 3.85e4 ns T2 = 8.1e3 ns,
q4 T1 = 3.58e4 ns T2 = 8e3 ns.
Thanx Tatjana!

### CODE REVIEW

**Comment by alecandido** [comment]
@stavros11 I tried to rebase on `main`, since there were commits not strictly related to the platinum.

In particular, before even the iqm5q was reported to be different from `main`. However, it seems that most of the commits were also affecting the platinum, so they might be slightly changed, but they have been more or less left unchanged.

**Comment by alecandido** [comment]
@biankawolo in case you have problems pulling, do this:
```sh
git pull --rebase
```
in place of the usual `git pull`.

**Review by stavros11 — APPROVED** [review]
Qubits 0, 2 seem to work. I am not sure what is wrong with qubit 4, but maybe we could merge regardless, in order to have something on main.

* Qubit flux: http://login.qrccluster.com:9000/h7pUz3J-Q1-xmHJ-3bZfuw==
* Rabi: http://login.qrccluster.com:9000/uJyamyiOSUOtefQhID1eug==
* Rabi length: http://login.qrccluster.com:9000/a9yX29j0S_6WrWCrck79Eg==
* Single shot: http://login.qrccluster.com:9000/eQua-Yh4TJGUy_g6-eMuMg==

**Review by alecandido — APPROVED** [review]

File: qw5q_platinum/platform.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    load_instrument_settings,
+    load_qubits,
+    load_runcard,
+    load_settings,
+)
+
+FOLDER = pathlib.Path(__file__).parent
+
+
+def create():
```
Comment (DavidSarlle) [inline qw5q_platinum/platform.py:18]: I think this not controlled by qblox cluster, right? Is QM

Comment (alecandido) [reply qw5q_platinum/platform.py:18]: Exactly, you can read it from the imports:
https://github.com/qiboteam/qibolab_platforms_qrc/blob/4a356902fff4f6a416f8b1072d8db19c5879c8dc/qw5q_platinum/platform.py#L4

Comment (DavidSarlle) [reply qw5q_platinum/platform.py:18]: yes I know, I was just being nice :) In the docstring says:

    """QuantWare 5q-chip controlled using qblox cluster.
    Args:
        runcard_path (str): Path to the runcard file.
    """
    
    So, probably we can change it

Comment (stavros11) [reply qw5q_platinum/platform.py:18]: Indeed, the docstring was wrong, thanks @DavidSarlle. I fixed this in be0a77e6e0f205a61d84e8617b7d7ebc32d33b4f and added some details about the current calibration.

**Pull request merged**